src/Microsoft.AspNet.Http/SessionCollection.cs

```diff
@@ -1,74 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections;
-using System.Collections.Generic;
-using Microsoft.AspNet.Http.Features;
-
-namespace Microsoft.AspNet.Http.Internal
-{
-    public class SessionCollection : ISessionCollection
-    {
-        private readonly ISession _session;
-
-        public SessionCollection(ISession session)
-        {
-            _session = session;
-        }
-
-        public byte[] this[string key]
```

Comment:
We discussed about this earlier and concluded that users should use the extensions rather than the indexer as its just a byte array.